cernopendata/modules/theme/bundles.py

```diff
@@ -30,25 +30,25 @@
 from invenio_assets import NpmBundle
 
 js = NpmBundle(
-    'node_modules/angular/angular.js',
+    # 'node_modules/angular/angular.js',
```

Comment:
What about the Angular directives? We have 'ng-if` and friends in many Jinja places. If we "close" the issue and not only "address" it, then I guess we'd have to remove all `ng-if`, `ng-controller` and friends from the pages that are declared working.  Is that OK already for all the checked endpoints? 

P.S. For `/search`, `/visualise/...` etc this will be naturally addressed elsewhere.

Reply:
After scanning through the code, I only found Angular directives used in `/search` related templates:
- `cernopendata/templates/cernopendata_records_ui`
- `cernopendata/templates/cernopendata/search.html`

and also in components used by `/search` under the `/cernopendata/static`. So I guess it could be addressed while converting to `react`?